src/test/components/__snapshots__/TooltipMarker.test.js.snap

```diff
@@ -2286,6 +2286,20 @@ exports[`TooltipMarker renders tooltips for various markers: GCSlice-17.5 1`] =
       :
     </div>
     1
+    <div
+      class="tooltipLabel"
+    >
+      Phase mark
+      :
+    </div>
+    10ms
+    <div
+      class="tooltipLabel"
+    >
+      Phase wait_background_thread
+      :
+    </div>
+    0.000ms
```

Comment:
Thanks for spotting this, it does look wrong and I had to dig into it to figure out what's going on.

There weren't any other phases here that are greater than 0. The ones you see are from a preceding test. This particular output comes from a test with a very short list of phases: 100046µs for `mark` and 0µs for `wait_background_thread`. So the only difference here is that the original code dropped any phase with zero time. I don't know if that was intentional; it happened because it computes a threshold and only keeps phases with time more than that threshold. I'm unconditionally keeping the top N.

I'm not sure which is better. I believe in practice the engine won't emit zero-time phases, so I don't think it matters in practice. But if it *were* to output a zero-time phase, I think I'd rather it drop it; it's not useful information, and the only point of this stuff is to try to narrow the displayed information down to the useful subset. So I think I'll make it drop zero time phases.